homeassistant/components/hassio/__init__.py

```diff
@@ -149,10 +150,12 @@
 SERVICE_RESTORE_FULL = "restore_full"
 SERVICE_RESTORE_PARTIAL = "restore_partial"
 
+VALID_ADDON_SLUG = vol.Match(re.compile(r"^[-_.A-Za-z0-9]+$"))
```

Comment:
An add-on slug should be exactly 1 `.` and with text on both sides. This validation would also allow `..`.

Reply:
I think `.` are fine also `....` (make no sense) as long you not can use `/`